tests/jest/unit/core/common/assets/js/api/core/store.test.js

```diff
@@ -0,0 +1,68 @@
+import Store from 'elementor-common/api/core/store';
+import { createSlice, combineReducers } from '@reduxjs/toolkit';
+
+jest.mock( '@reduxjs/toolkit', () => ( {
+	...jest.requireActual( '@reduxjs/toolkit' ),
+	combineReducers: jest.fn( () => 'combine-reducers' ),
+	configureStore: jest.fn( () => ( {
+		replaceReducer: jest.fn(),
+	} ) ),
+} ) );
+
+// Arrange.
+let store, slice1, slice2;
+
+beforeEach( () => {
+	store = new Store();
+
+	slice1 = createSlice( {
+		initialState: {},
+		reducers: {},
+		name: 'slice-1',
+	} );
+
+	slice2 = createSlice( {
+		initialState: {},
+		reducers: {},
+		name: 'slice-2',
+	} );
+} );
+
+afterEach( () => {
+	jest.clearAllMocks();
+} );
```

Comment:
`beforeEach` and `afterEach` should live inside the `describe` function (also the slice and store variables)

Reply:
Done